holidays/countries/turkmenistan.py

```
@@ -0,0 +1,69 @@
+from holidays.calendars.gregorian import JAN, MAR, MAY, SEP, OCT, DEC
```

Ruff (0.8.2)

`holidays.calendars.gregorian.JAN` imported but unused

Remove unused import

(F401)